common_bugs/setup_bugs/README.md

```diff
@@ -10,3 +10,10 @@
 * Test this by having student type `source ~/.bashrc` and running `pry`. If `pry` works here, it is a zshell error.
 
 * The easiest fix is to switch back to `bash`. First have the student `cd` to root then type `chsh -s /bin/bash`. Opening a new terminal tab should show `bash` and not `zsh` at the top, and `pry` should now open successfully.
+
+## Mac VSCode terminal runninng on `sh` and unable to switch to `bash`
+
+* This is an issue when VSCode does not allow the terminal to switch to run on bash 
+
+* The easiest fix is to check `settings.json` and make sure they are have 
+` "terminal.integrated.automationShell.osx": ""` and not other terminal defaults(example: `"terminal.integrated.shellArgs.linux"` )
```

Comment:
can you include a sentence in here explaining how you came across this error.  saying something along the lines of "you can come across this error when students byebug works on the external terminal but not on internal vscode terminal ".